content/silver/ref/expr/pattern-matching.md

```diff
@@ -96,21 +122,20 @@ case d(c()) of
 end
 ```
 
-If we rearrange the rules in this last `case` expression, we get the same result.
-Matching `d(c())` against `b(_, _)` fails, so we forward and try again.
-Matching `a(c())` against `b(_, _)` also fails.
-Since we have no more forwards to try matching, we move on to the next pattern and try matching `d(c())` against it.
-When this fails, we forward again and find a match.
+If we rearrange the rules in this last `case` expression, we get the same result:
```

Comment:
Maybe say "If we rearrange the rules in the last `case` expression to form the following one, we get the same result:"